frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx

```diff
@@ -90,7 +90,22 @@ export function SummaryTable({
         })
         columns.push({
             key: 'exposure',
-            title: 'Exposure',
+            title: (
+                <div className="inline-flex items-center space-x-1">
+                    <div className="">Exposure</div>
+                    <Tooltip
+                        title={
+                            <div>
+                                The number of people who were exposed to this variant. By default, this is measured by
```

Comment:
```suggestion
                                The number of users who were exposed to this variant. By default, this is measured by
```

While I think "users" sounds like drug users, I think it's more consistent with the rest of the app.